File: app/src/main/java/com/legue/axel/lolsummonertool/network/response/match/MatchlistDto.java
package com.legue.axel.lolsummonertool.network.response.match;

import java.util.List;

public class MatchlistDto {

    public List<MatchReferenceDto> matches;
    public int totalGames;
    public int startIndex;
    public int endIndex;

    public MatchlistDto() {
    }


    public MatchlistDto(List<MatchReferenceDto> matches, int totalGames, int startIndex, int endIndex) {
        this.matches = matches;
        this.totalGames = totalGames;
        this.startIndex = startIndex;
        this.endIndex = endIndex;
    }
}

File: app/src/main/java/com/legue/axel/lolsummonertool/network/response/match/MatchlistDto.kt
package com.legue.axel.lolsummonertool.network.response.match

data class MatchlistDto(
        var matches: List<MatchReferenceDto>? = null,
        var totalGames: Int = 0,
        var startIndex: Int = 0,
        var endIndex: Int = 0)
